src/procedural.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace procedural;
```

Comment:
Afaik there's no relation between namespaces, case types and directories content in any PSR-0/PSR-4 standard. The only constraint is to be consistent with the [`composer.json` definition of the repo](https://github.com/jhabdas/learn-kahlan/blob/403a193ca54caae828918f70e6f3e5b470614534/composer.json#L3-L10)